server/status.go

```diff
@@ -92,6 +92,10 @@ const (
 	// healthEndpoint is a shortcut for local details, intended for use by
 	// monitoring processes to verify that the server is up.
 	healthEndpoint = "/health"
+
+	// prometheusEndpoint is the prometheus metrics endpoint, not to be
+	// confused with the built-in cockroach metrics.
+	prometheusEndpoint = "/metrics"
```

Comment:
Does prometheus require the path `/metrics`, or could this be something like `/prometheus-metrics`? Or could this potentially replace our built-in metrics endpoint?